### DISCUSSION TITLE
Cannot save Preset Descriptions

### DISCUSSION THREAD

**Discussion by Bluc69** [Builty/TonexOneController#33] [General]
I'm using a Waveshare 4.3" B LCD board with firmware release 1.0.3.2 and I cannot save the preset descriptions.
I tried several times to write a description and save it by pressing the green tick image, but if I change the preset and go back to the one I modified the description for, I can no longer find it saved. 
Conversely, if I change the amplifier image and press the green arrow, the change is saved permanently.
What am I doing wrong?

**Comment by Builty** [comment]
Hi. On the keyboard that pops up, make sure you press the enter button on it, and not just click away to close the keyboard.
This part is not quite perfect, but I haven't found a better way yet.

**Reply by Bluc69** [reply]
Hi,
thank you, it works!
Maybe it would be helpful to put this explanation in the "Usage" paragraph.
